appservice/src/createAppService/AppServicePlanRedundancyStep.ts

```diff
@@ -46,25 +46,37 @@ export class AppServicePlanRedundancyStep extends AzureWizardPromptStep<IAppServ
             'eastus2euap',
         ];
 
+        location = location.replace(/\s/, "").toLowerCase();
```

Comment:
I don't see this in the previous code. What did you run into that prompted you to make this change?

Reply:
The data that gets stored for the location seems to be different than the name.  So the name of our quick pick is usually something like `East US` but will be stored as 'eastus'.  I basically wanted to enable being able to check locations by passing in either type of field, not just the latter since we typically use the name of the pick as part of the inputs list of testing

Reply:
Take a look at `LocationListStep`. Could you use `locationMatchesName` or `generalizeLocationName` for the same task?

Reply:
`generalizeLocationName` would be perfect, but it's not exported, so would need an update to azure package and release.

Reply:
Added a todo for this item since not currently expored